README.md

```diff
@@ -49,13 +49,13 @@ See the [compile.md](doc/guides/compile.md) file.
 
 There are two sets of documentation in mruby: the mruby API (generated by yard) and C API (Doxygen)
 
-To build both of them, simply go
+To build both of them, simply navigate:
```

Comment:
I am not an English native, but the word "navigate" makes me feel a bit awkward. Is it natural for command-line interaction?